sklearn/linear_model/ridge.py

```diff
@@ -984,7 +985,7 @@ class RidgeClassifierCV(LinearClassifierMixin, _BaseRidgeCV):
 
     Parameters
     ----------
-    alphas : numpy array of shape [n_alphas]
+    alphas : numpy array or tuple of length [n_alphas]
```

Comment:
Sorry let's try to find a common agreement. The original was 
`- alphas : numpy array of shape [n_alphas]`  
Can I replace it with: `array-like of shape [n_alphas]` ?